dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/email/EmailContact.kt

```diff
@@ -0,0 +1,45 @@
+package org.dataland.datalandcommunitymanager.model.email
+
+import com.mailjet.client.transactional.SendContact
+import com.mailjet.client.transactional.TransactionalEmail
+
+/**
+ * A class to represent an email contact
+ */
+data class EmailContact(
+    val emailAddress: String,
+    val name: String? = null,
+) {
+    /**
+     * Defines whether to hide the requester's name in the email or to show it
+     */
+    fun toMailjetSendContact(): SendContact {
+        return if (name == null) SendContact(emailAddress) else SendContact(emailAddress, name)
+    }
+}
+
+/**
+ * Uses a list of EmailContact objects for the build of a TransactionalEmail
```

Comment:
The descriptions could be more speaking, right now they are all identical.

Reply:
I updated them and made them more precise

Reply:
Thank you for changing them. I will close this now